internal/pkg/service/common/etcdop/op/atomic_test.go

```diff
@@ -141,8 +141,11 @@ func TestAtomicOp(t *testing.T) {
 		},
 		{
 			// Read Phase gets a keys prefix, but before the Write Phase, an existing key is deleted in the prefix.
-			Name:           "GetPrefix_DeleteKey",
-			Prepare:        putTwoKeys,
+			Name: "GetPrefix_DeleteKey",
+			Prepare: func(t *testing.T, client etcd.KV) []op.Op {
+				t.Skip("not implemented/impossible: we cannot check that a key was deleted from the prefix")
+				return putTwoKeys(t, client)
+			},
 			ReadPhase:      getPrefix,
```

Comment:
But there is a problem, one test-case is not covered by the logic.

I have a prefix "key"` with keys "key/1", "key/2".

I read the prefix in get phase, so write phase something depends on the all prefix values.

But `BrakingChange` removed `"key/2"`.

Write Phase conditions are MET, because:
- There is at least one key in the prefix `MOD > 0`.
- No key has been modified `MOD < read phase revision + 1`.

In the etcd is missing a `COUNT` operation in TXN IF condition, that would help us solve the issue.
https://github.com/Revolution1/etcd3-py/issues/76

You can only check, the prefix is empty, via `MOD == 0`.